### PR TITLE
Fix save payment method checkbox alignment in the CC form when upe is disabled

### LINKED ISSUE #2000: "Save Payment Method" checkbox not aligned properly with credit card form when UPE is disabled.
Borrowing (and paraphrasing) from @FangedParakeet as this came up in his review of #1999 :

With the UPE disabled and the Twenty Twenty-One theme activated, the saved payment method checkbox is sitting in a slightly odd, offset position.

Wonky saved payment checkbox
_Slightly offset saved payment method checkbox_

After a light inspection, I discovered that this is actually because the checkbox is [displayed outside of the `wc-payment-form` `fieldset` element](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/includes/class-wc-gateway-stripe.php#L255-L260) with the UPE disabled, while with the UPE enabled it sits snugly inside this field group. As a result of this, the padding that is applied to the rest of the payment elements is not applied to the checkbox.

No padding for checkbox
_Exiled checkbox has no padding_

Here's what it looks like inside the `fieldset`--perhaps a little cramped, because I have not applied any additional styling, but at least now aligned.

Padding for checkbox
_Checkbox promoted to sibling now has padding_

**To Reproduce**
1. Disable UPE
2. Activate the Twenty Twenty-One theme
2. Add a product to the cart and go to checkout
4. See the error.

**Expected behavior**
The checkbox should be aligned with the credit card form in this theme.

**Environment (please complete the following information):**
 - WordPress Version: 5.8.1
 - WooCommerce Version: 5.7.1
 - Stripe Plugin Version: 5.6.0
 - Browser [e.g. chrome, safari] and Version: Chrome 93.0.4577.82

### PR DESCRIPTION
**Pull request by rafaelzaleski**
# Changes proposed in this Pull Request:

fix #2000 

This PR makes the tag hierarchy and positioning of the "Save Payment Method" button match when UPE is enabled or disabled. Previously, the button was shown inside a fieldset when UPE was enabled; and outside of the fieldset when it was disabled.

# Testing instructions

## Test with UPE disabled

1. Activate the Twenty Twenty-One theme
2. Disable UPE
3. Add a product to the cart, and go to the checkout page.
4. The checkbox to save the payment method should be aligned with the credit card fields.

Screen Shot 2021-10-08 at 15 26 37

## Test with UPE enabled

1. Activate the Twenty Twenty-One theme
2. Enable UPE
3. Add a product to the cart, and go to the checkout page.
4. The checkbox to save the payment method should be aligned with the credit card fields.

Screen Shot 2021-10-08 at 15 27 52

### CODE REVIEW

File: includes/class-wc-gateway-stripe.php [hunk 1]
```diff
@@ -316,7 +316,6 @@ public function elements_form() {
 
 			<!-- Used to display form errors -->
 			<div class="stripe-source-errors" role="alert"></div>
-			<br />
```
Comment (reykjalin) [inline includes/class-wc-gateway-stripe.php:308]: Out of curiosity; is there a reason this `<br />` element had to be removed? I'm not aware of any reason it has to stay either, just curious 🙂

**Review by reykjalin — APPROVED** [review]
Looks good, tests well 🎉 

I had one minor comment, but it's not a blocker for merging 🙂 

🚀

File: includes/class-wc-gateway-stripe.php (hunk 1 above)
Comment (rafaelzaleski) [reply includes/class-wc-gateway-stripe.php:308]: @reykjalin It was causing excessive space between the CVV field and the checkbox. I couldn't find a reason for it too.

Comment (reykjalin) [reply includes/class-wc-gateway-stripe.php:308]: Makes sense!

**Pull request merged**